Abb 79 156 174

DenisNosnitsin1970 opened the pull request:
minor changes

eu-genia commented on ParisBNF/abb/BNFabb156.xml:
```diff
@@ -419,7 +419,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                        <item xml:id="a5">
                           <locus from="67rb" to="67v"/>
                           <desc type="GuestText" corresp="LIT2332serata" cert="low">A short Śǝrʿāta beta krǝstiyān text in Gǝʿǝz, 
-                             probably incomplete, possibly somehow related to <ref target="#a1 #a2 #3">Additional notes 1, 2 or 3</ref>.</desc>
```
why not links? they were wrong, it should be
`Additional notes <ref target="#a1">1</ref>, <ref target="#a2">2</ref> or <ref target="#a3">3</ref>`

DenisNosnitsin1970 replied:
ok

DenisNosnitsin1970 commented:
I have added the description of Abb 190. It is a difficult piece and there are some issues which I have left unsolved (Mss from the BLib that would be needed for that, are not available for the moment). I added also stubs for 6 mss described during the workshop at Taqot Maryam.

eu-genia commented on ParisBNF/abb/BNFabb190.xml:
```diff
@@ -0,0 +1,411 @@
+
+                    <msItem xml:id="p1_i1">
+                       <locus from="1ra" to="2vb"/>
+                      <title type="incomplete" cert="medium">A story about the finding of The Holy Cross featuring Empress Helena and Cyriac (the beginning is mising)</title>
```
missing

I would probably put the beginning is missing as a note, or write (acephalous)

eu-genia commented on ParisBNF/abb/BNFabb190.xml:
```diff
@@ -0,0 +1,411 @@
+                       
+                        <note>The paper guard covers the beginning of the lines on 
+                           <locus target="#3ra"/>. Written by several hands, the leaves are probably in disorder. 
+                           <bibl><ptr target="bm:ted1988Exeget"/><citedRange unit="page">461</citedRange>
```
bm:Cowley1988Exeget?

DenisNosnitsin1970 commented on ParisBNF/abb/BNFabb190.xml:
```diff
@@ -89,14 +89,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             በአሕማር፡ ብዙኅ፡ ከመ፡ <sic resp="DN">ይቅልዎ፡</sic> ወያጥፍኡ፡ ሀገሮ፡ ወከበብዋ፡ ለሀገሩ፡</incipit>
                        <explicit xml:lang="gez">ለንግሥትኒ፡ እሌኒ፡ ሰብሕዋ፡ ወአክበርዋ፡ በብዙኅ፡<sic resp="DN">ከብር፡ </sic> ምእመናን፡ እስመ፡ ወሀበቶሙ፡ ሕይወተ፡ 
                           ዘኢየኃልቅ፡ ወኢየኃልፍ፡ ለዓ<cb n="b"/> ለመ፡ ዓለም፨</explicit>
+                       <note>The beginning is mising.</note>
```
Oh, sorry.
